scripts/savu_mod/savu_mod.py

```diff
@@ -65,6 +71,16 @@ def load_process_list(process_list):
     sc._ref(content, '* -n')
     return content
 
+def timestamp_file(filename):
+    """ Return a filename with a timestamp
+
+    :param filename: filename input
+    :return: t_filename, the filename with a timestamp
+    """
+    cur_time = datetime.now().strftime("%Y_%m_%d-%Hh_%Mm_%Ss_%f")
```

Comment:
changed the format since the presence of colon (probably) was resulting in an error while saving

Reply:
Thanks! It did seem to be working for me, but I'm glad it's working in your case too